Assets/MRTK/SDK/Features/Utilities/Solvers/HandConstraint.cs

```diff
@@ -536,7 +536,7 @@ private static bool IsApplicableController(IMixedRealityController controller)
             return controller.ControllerHandedness != Handedness.None;
         }
 
-        private static IMixedRealityController GetController(Handedness handedness)
+        protected static IMixedRealityController GetController(Handedness handedness)
```

Comment:
Since this is now protected, please add comment describing what the method does.

Reply:
Can do!